advocacy_docs/dev-guides/deploy/windows.mdx

```diff
@@ -0,0 +1,149 @@
+
+!!! Tip
+
+You can find a list of available PostgreSQL versions by using WinGet's `search` command:
+
+```
+winget search PostgreSQL.PostgreSQL
+___OUTPUT___
+Name          Id                       Version Source
+------------------------------------------------------
+PostgreSQL 10 PostgreSQL.PostgreSQL.10 10      winget
+PostgreSQL 11 PostgreSQL.PostgreSQL.11 11      winget
+PostgreSQL 12 PostgreSQL.PostgreSQL.12 12.20-1 winget
+PostgreSQL 13 PostgreSQL.PostgreSQL.13 13.16-1 winget
+PostgreSQL 14 PostgreSQL.PostgreSQL.14 14.13-1 winget
+PostgreSQL 15 PostgreSQL.PostgreSQL.15 15.8-1  winget
+PostgreSQL 16 PostgreSQL.PostgreSQL.16 16.4-1  winget
+PostgreSQL 9  PostgreSQL.PostgreSQL.9  9       winget
+```
+
+Installers for release candidate versions of PostgreSQL are also available on EDB's website at https://www.enterprisedb.com/downloads/postgres-postgresql-downloads
+!!!
+
+!!! SeeAlso "Further reading"
+For more fine-grained control, you can also download and run the installer interactively by following the instructions in [Installing PostgreSQL on Windows](https://www.enterprisedb.com/docs/supported-open-source/postgresql/installing/windows/). 
+!!!
+
+## Add PostgreSQL commands to your path
+
+PostgreSQL comes with [several useful command-line tools](https://www.postgresql.org/docs/current/reference-client.html) for working with your databases. For convenience, you'll probably want to add their location to your path.
+
+1.  Open the System Properties control panel and select the Advanced tab (or run `SystemPropertiesAdvanced.exe`)
+2.  Activate the "Environment Variables..." button to open the envornment variables editor
+3.  Find the `Path` variable under the "System variables" heading, and click "Edit..."
+4.  Add the path to the bin directory of the PostgreSQL version you installed, e.g.  `C:\Program Files\postgresql\16\bin\` (where *16* is the version of PostgreSQL that you installed).
+
+!!! Warning "This only affects new command prompts"
+If you have a command prompt open already, you'll need to close and reopen in order for your changes to the system path to take effect.
+!!!
+
+## Verifying your installation
+
+If the steps above completed successfully, you'll now have a PostgreSQL service running with the default database and superuser account. Let's verify this by connecting, creating a new user and database, and then connecting using that user.
+
+### Connect with psql
+
+Open a new command prompt, and run
+
+```
+psql -U postgres
+___OUTPUT__
+Password for user postgres:
+
+psql (16.4)
+WARNING: Console code page (437) differs from Windows code page (1252)
+         8-bit characters might not work correctly. See psql reference
+         page "Notes for Windows users" for details.
+Type "help" for help.
+
+postgres=#
+```
+
+When prompted, enter the default password (`postgres`). 
+
+!!! Warning
```

Comment:
"Warning" strikes me as a little more alarming than what is required here. Not sure though.

Reply:
could go with "Danger"... ;-P
I shortened it a bit; I think at least *some* warning is warranted; as obvious as this seems, not a year goes by without some big fiasco caused by folks putting DBs in production with the default password intact...

Reply:
cool cool
that makes sense